entrypoint.sh

```diff
@@ -22,7 +22,7 @@ echo "Setting up git machine..."
 git_setup
 
 echo "Forcing tag update..."
-git tag -a latest -m "This tag has been auto-generated by latest-tag" "${GITHUB_SHA}" -f
+git tag -a latest -m $INPUT_INTRODUCTION "${GITHUB_SHA}" -f
```

Comment:
I think we need to put some quotes around it, don't we? Like: `"$INPUT_INTRODUCTION"`